sardana_albaem/ctrl/Albaem2OneDCtrl.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/env python
+import socket
+import time
+import datetime
+from threading import Lock
+
+from sardana import State, DataAccess
+from sardana.pool import AcqSynch
+from sardana.pool.controller import OneDController, Type, Access, \
+    Description, Memorize, Memorized, NotMemorized, FGet, FSet, DefaultValue
+from sardana.sardanavalue import SardanaValue
+from functools import wraps, partial
+import six
+
+__all__ = ['Albaem2OneDCtrl']
+
+TRIGGER_INPUTS = {'DIO_1': 0, 'DIO_2': 1, 'DIO_3': 2, 'DIO_4': 3,
```

Comment:
where is this used?

Reply:
i based it on the ct controller and took it from there, should i remove it?

Reply:
If it is not used, better remove it